File: 01 Capitulo Layout/PrimeiroApp/app/src/main/java/br/edu/unicesumar/primeiroapp/MainActivity.java
package br.edu.unicesumar.primeiroapp;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.coordinator_main);


        Log.d("categoria", "mensagem de debug");
        Log.i("categoria", "mensagem de informação");
        Log.w("categoria", "mensagem de aviso");
        Log.e("categoria", "mensagem de erro");
        Log.e("categoria", "mensagem de erro", new RuntimeException("erro"));

    }
}

File: Kotlin/99 - Código fonte das aulas gravadas/meuprimeiroaplicativo/app/src/main/java/br/com/uniciv/meuprimeiroaplicativo/MainActivity.kt
package br.com.uniciv.meuprimeiroaplicativo

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.util.Log

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Log.d("categoria", "mensagem de debug")
        Log.i("categoria", "mensagem de informacao")
        Log.w("categoria", "mensagem de aviso")
        Log.e("categoria", "mensagem de erro")


    }
}
